crates/cairo-coverage/tests/helpers/test_project.rs

```diff
@@ -0,0 +1,105 @@
+use anyhow::{Context, Result};
+use assert_fs::fixture::PathCopy;
+use assert_fs::TempDir;
+use camino::Utf8PathBuf;
+use snapbox::cmd::{cargo_bin, Command as SnapboxCommand};
+use std::fs;
+use std::path::PathBuf;
+
+pub struct TestProject {
+    dir: TempDir,
+}
+
+impl TestProject {
+    pub fn new(test_project_name: &str) -> Result<Self> {
+        let dir = TempDir::new().context("Failed to create a temporary directory")?;
+        dir.copy_from(
+            format!("tests/data/{test_project_name}/"),
+            &["*.toml", "*.cairo"],
+        )
+        .context("Failed to copy project files to the temporary directory")?;
+        Ok(Self { dir })
+    }
+
+    fn generate_trace_files(self) -> Self {
+        SnapboxCommand::new("snforge")
+            .arg("test")
+            .arg("--save-trace-data")
+            .current_dir(&self.dir)
+            .assert()
+            .success();
+        self
+    }
+
+    fn find_trace_files(&self) -> Result<Vec<String>> {
+        let trace_path = self.dir.path().join("snfoundry_trace");
+        fs::read_dir(&trace_path)
+            .context("Failed to read the directory for trace files")?
+            .map(|entry| {
+                entry
+                    .context("Failed to read a directory entry")
+                    .map(|e| e.path().display().to_string())
+            })
+            .collect()
+    }
+
+    fn output_lcov_path(&self) -> PathBuf {
```

Comment:
With the current implementation, it seems quite tricky to test anything using the `--output-path` flag. Maybe it would be better to modify it to be more flexible?

Reply:
Change that and added test for this flag